App/AlleSeller/AlleSeller/Assets.xcassets/searchGlyphIcon.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
```

Comment:
Why not SFSymbols? :)

Reply:
Because on design it is different than SF Symbols search one :)